### PR TITLE
ZCS-3017 Add separate attributes for NG modules initialisation

### PR DESCRIPTION
**Pull request by rupalid**
ZCS-3017
Added 2 new attributes for HSM, Admin and Backup modules. We already have attributes for Admin and Mobile. The one for Admin was deprecated .We now have the following attributes
zimbraNetworkAdminNGEnabled
zimbraNetworkMobileNGEnabled
zimbraNetworkBackupNGEnabled
zimbraNetworkHSMNGEnabled

### CODE REVIEW

File: store/conf/attrs/zimbra-attrs.xml [hunk 1]
```diff
@@ -9379,13 +9379,14 @@ TODO: delete them permanently from here
...
 <attr id="2118" name="zimbraNetworkMobileNGEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.8.0" >
-  <globalConfigValue>TRUE</globalConfigValue>
+  <globalConfigValue>FALSE</globalConfigValue>
   <desc>Whether to enable zimbra network new generation mobile sync module.</desc>
 </attr>
 
-<attr id="2119" name="zimbraNetworkAdminEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.8.2">
-  <globalConfigValue>TRUE</globalConfigValue>
+<attr id="2119" name="zimbraNetworkAdminEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.8.2" deprecatedSince="8.8.5">
+  <globalConfigValue>FALSE</globalConfigValue>
```
Comment (Prashantsurana) [inline store/conf/attrs/zimbra-attrs.xml]: Minor: no need to change default value as it is deprecated.

File: store/conf/attrs/zimbra-attrs.xml [hunk 2]
```diff
@@ -9586,15 +9587,20 @@ TODO: delete them permanently from here
   </desc>
 </attr>
 
-<attr id="2126" name="zimbraNetworkHSMNGEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.8.3">
+<attr id="2129" name="zimbraNetworkHSMNGEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.8.5">
```
Comment (Prashantsurana) [inline store/conf/attrs/zimbra-attrs.xml]: I am not sure if changing id will break upgrade or not. Can you please test 8.8.3 to 8.8.5 upgrade.

Comment (Prashantsurana) [reply store/conf/attrs/zimbra-attrs.xml]: If this is new attribute and not yet released, it should be fine.

File: store/conf/attrs/zimbra-attrs.xml (hunk 1 above)
Comment (rupalid) [reply store/conf/attrs/zimbra-attrs.xml]: Fixed

File: store/conf/attrs/zimbra-attrs.xml [hunk 3]
```diff
@@ -9372,12 +9372,13 @@ TODO: delete them permanently from here
 <desc>outgoing sieve script defined by admin (not able to edit and view from the end user) applied after the end user filter rule</desc>
 </attr>
 
-<attr id="2117" name="zimbraNetworkModulesNGEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.8.0">
+<attr id="2117" name="zimbraNetworkModulesNGEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.8.0" deprecatedSince="8.8.3">
```
[the remark this answers is not in the document]
Comment (rupalid) [reply store/conf/attrs/zimbra-attrs.xml]: Fixed

File: store/conf/attrs/zimbra-attrs.xml (hunk 2 above)
Comment (rupalid) [reply store/conf/attrs/zimbra-attrs.xml]: This is a new attribute, so it will be fine.

**Pull request closed without merging**